README.md

```diff
@@ -1,69 +1,405 @@
-- [Airgap](https://airgap.it) - Offline secure wallet 
-- [Base App](https://base.aepp.com) - Webapp wallet and dAepp directory
-- [Waellet](https://waellet.com) - Metamask for Aeternity
-
-## Web tools 
-- [Frontend](https://mainnet.aeternal.io) - Frontend / explorer (Mainnet) 
-- [Frontend](https://testnet.aeternal.io) - Frontend / explorer (Testnet) 
-- [AEknow](https://aeknow.org/) - Blockchain explorer (Mainnet)
-
-## SDKs 
-- [Javascript](https://github.com/aeternity/aepp-sdk-js) (Official)
-- [Python](https://github.com/aeternity/aepp-sdk-python) (Official)
-- [Go](https://github.com/aeternity/aepp-sdk-go) (Official)
-- [Elixir](https://github.com/aeternity/aepp-sdk-elixir) (Official)
-- [Java](https://github.com/kryptokrauts/aepp-sdk-java) (Community)
-
-## Developer Tools
-- [AEproject](https://github.com/aeternity/aepp-aeproject-js) - Truffle for Aeternity 
-- [Contracts editor](http://fireeditor.nikitafuchs.de) (Fire editor) - Online contract editor
-- [Testnet faucet](https://testnet.faucet.aepps.com) - Top-up your account on the Testnet network
-
-## Official hosted services 
-- [mainnet.aeternity.io](https://mainnet.aeternity.io/v2/status) - Mainnet API Gateway (node only)
-- [mainnet.aeternal.io](https://mainnet.aeternal.io) - Mainnet gateway (with middleware)
-- [testnet.aeternity.io](https://testnet.aeternity.io/v2/status) - Testnet API Gateway (node only)
-- [testnet.aeternal.io](https://testnet.aeternal.io) - Testnet gateway (with middleware)
-- [compiler.aepps.com](https://compiler.aepps.com) - Hosted Sophia compiler
-
-## Tutorials and Examples
-
-There is an extensive list of [tutorials](https://github.com/aeternity/tutorials) to get started, the following are instead sample applications 
-
-- [Smart oracles in Elixir](https://github.com/DanielaIvanova/smart_oracle) - Oracles with Elixir
-- [Contracts in Java](https://github.com/kryptokrauts/contraect-showcase-maven) - Interact with contracts with Java
-- [4evaAens](https://github.com/u2467/4evaAens) - Auto bidding for aens (Javascript) - from Hack&Play/11.2019
-- [4evaAens](https://github.com/DanielaIvanova/forever_aens) - Auto bidding for aens (Elixir) - from Hack&Play/11.2019
-- [AENS Marketplace](https://github.com/DanielaIvanova/name_marketplace) - Marketplace for AENS (Elixir) - from Hack&Play/11.2019
-- [AENS Marketplace](https://github.com/u2467/aens-marketplace) - Marketpalce for AENS (Python/Sophia) - from Hack&Play/11.2019 
-- [TypeScript demo](https://github.com/gcofficial/ae-typescript-demo) - Use the JS SDK with TypeScript
+
+### Browser extensions
+- Superhero Wallet - The most advanced aeternity wallet with additional features for Superhero.com.
+    - [Brave / Chrome](https://chrome.google.com/webstore/detail/superhero/mnhmmkepfddpifjkamaligfeemcbhdne?hl=en)
+    - [Firefox](https://addons.mozilla.org/en-US/firefox/addon/superhero-wallet/)
+- [Waellet](https://github.com/waellet/waellet) - Currently not actively developed.
+
+### Cloud & custodial wallets
```

Comment:
what about the chinese web wallet, don't remember the name

Reply:
Not sure about that one. Could also be added later on. So AEBox is already included in the mobile app section and I don't think there exists a web wallet for it.

If you mean https://aeasy.io -> I added it under development framework as it describes itself to be.

And WeTrue is under the applications list as social aepp.